scripts/coverage.js

```diff
@@ -10,7 +10,7 @@ var Mocha = require('mocha'),
     Instrumenter = require('istanbul').Instrumenter,
     Report = require('istanbul').Report,
     Collector = new require('istanbul').Collector,
-    sourcefiles = ['index.js', 'extensions.js', 'render.js'],
```

Comment:
Ah correct, sorry in my phone :+1:
On 10 Jun 2016 2:58 PM, "Nick Schonning" <notifications@github.com> wrote:

> In scripts/coverage.js
> <https://github.com/sass/node-sass/pull/1583#discussion_r66561805>:
>
> > @@ -10,7 +10,7 @@ var Mocha = require('mocha'),
> >      Instrumenter = require('istanbul').Instrumenter,
> >      Report = require('istanbul').Report,
> >      Collector = new require('istanbul').Collector,
> > -    sourcefiles = ['index.js', 'extensions.js', 'render.js'],
>
> This is just the list of files to produce the coverage file for. It looks
> like the lowlevel.js is a consumer of the coverage.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sass/node-sass/pull/1583/files/491c3a4e96cd0df1e5dc91bd10b8b91d25351ce9#r66561805>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAjZWMNdHBxLAIfh1gTsofoSXXBHIqjvks5qKO7ggaJpZM4Iwm7Q>
> .
>